.github/workflows/maven.yml

```diff
@@ -21,34 +21,45 @@ on: [pull_request]
 
 jobs:
   linux:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         java: [ 8, 11, 14 ]
```

Comment:
There seems to be no java version 14 https://github.com/actions/setup-java.
There are only following versions available:
 8, 11, 16, 17
 
 Maybe we can replace 14 by 16 oder 17 .